### PR TITLE
Add buld script for Windows user.

### PR DESCRIPTION
**Pull request by Zaid-maker**
I have added this script because i am having issues with `rm` because it use in MAC PC's `del` is used to remove file same as `rm`  so yeah hope you guys like this.

### CODE REVIEW

**Comment by Zaid-maker** [comment]
By using del it solve my issue with compiling the project.

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "main": "./dist/index.js",
   "scripts": {
     "build": "npm ci && rm -rf ./dist && tsc",
+    "build2": "npm ci && del /f dist && tsc",
```
Comment (velddev) [inline package.json]: Kinda a bad fix, preferably use rimraf or something similar as a build tool

**Review by velddev — CHANGES_REQUESTED** [review]

**Comment by jpbberry** [comment]
Might honestly just remove the rm entirely. It was only there for when I was publishing from my computer rather than GitHub actions.

File: package.json [hunk 2]
```diff
@@ -1,10 +1,11 @@
 {
   "name": "@top-gg/sdk",
-  "version": "3.1.1",
+  "version": "3.1.2",
```
Comment (jpbberry) [inline package.json]: There's also no need to bump a change that only makes a difference to people who clone from GitHub.

**Review by jpbberry — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Zaid-maker) [reply package.json]: Means?

File: package.json (hunk 2 above)
Comment (Zaid-maker) [reply package.json]: ok

**Comment by Zaid-maker** [comment]
DONE

File: package.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
   "description": "Official Top.gg Node SDK",
   "main": "./dist/index.js",
   "scripts": {
-    "build": "npm ci && rm -rf ./dist && tsc",
+    "build": "npm ci && del /f dist && tsc",
```
Comment (jpbberry) [inline package.json]: I meant remove any RMing/DELing. So just `npm ci && tsc`

**Review by jpbberry — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (Zaid-maker) [reply package.json]: OHK hold on a sec.

**Review by jpbberry — APPROVED** [review]

**Comment by Zaid-maker** [comment]
Thank you so much ♥, GG's All checks have been passed.

**Review by velddev — APPROVED** [review]

**Pull request merged**